RecoBTag/Configuration/python/RecoBTag_EventContent_cff.py

```diff
@@ -41,9 +40,14 @@
         'keep *_pfCombinedInclusiveSecondaryVertexV2BJetTags_*_*',
         'keep *_pfCombinedMVABJetTags_*_*',
         'keep *_pfCombinedMVAV2BJetTags_*_*',
-	'keep *_pfCombinedSecondaryVertexSoftLeptonBJetTags_*_*',
-##	'keep *_pfCombinedSecondaryVertexSoftLeptonCtagLJetTags_*_*',
-        'keep *_inclusiveCandidateSecondaryVertices_*_*'
+        'keep *_pfCombinedSecondaryVertexSoftLeptonBJetTags_*_*',
+        ##	'keep *_pfCombinedSecondaryVertexSoftLeptonCtagLJetTags_*_*',
```

Comment:
is this still needed?
Somehow I understood that the "new" naming convention is Cvs*